content/02.main-text.md

```diff
@@ -242,10 +246,12 @@ We find that this asynchronous design with timestamps precise to the day is suit
 
 Manubot and its dependencies are free of charge and largely open source.
 It does rely on gratis services from two proprietary platforms: GitHub and Travis CI.
-Fortunately, lock-in to these services is minimal, and several substitutes already exist.
-Manubot provides a substantial step towards end-to-end document reproducibility, where every figure or piece of data in a manuscript can be traced back to its origin [@url:https://elifesciences.org/for-the-press/e6038800/elife-supports-development-of-open-technology-stack-for-publishing-reproducible-manuscripts-online] and is well suited for preserving provenance.
+Fortunately, lock-in to these services is minimal, and [several substitutes](#future) already exist.
+Manubot provides a substantial step towards end-to-end document reproducibility, where every figure or piece of data in a manuscript can be traced back to its origin [@url:https://elifesciences.org/for-the-press/e6038800/elife-supports-development-of-open-technology-stack-for-publishing-reproducible-manuscripts-online] and is well-suited for preserving provenance.
 For example, figures can be specified using versioned URLs that refer to the code that created them.
 In addition, manuscripts can be templated, so that numerical values or tables are inserted directly from the repository that created them.
+Figure @fig:contrib provides examples of templates.
+Phrases such as "{{total_commits}} Git commits" are written as `{{total_commits}} Git commits` so that the commit count can be automatically updated.
```

Comment:
I think you'll need to do:

```
{% raw %}`{{total_commits}} Git commits`{% endraw %}
```

http://jinja.pocoo.org/docs/2.10/templates/#escaping